### ISSUE TITLE
bug: Fix inconsistent page use of semantic HTML elements 

### ISSUE THREAD

**Issue by lilyndk** [WomenCodingCommunity/WomenCodingCommunity.github.io#200]
### Describe the bug
- Prominent vertical page section due to inconsistent page html
- Accessibility-wise, inconsistent use of semantic HTML elements can make it problematic for screen readers and other assistive technologies to properly interpret the content.
- Accessibility-wise, main content skip link is missing

### To Reproduce
- E.g. when inspecting the HTML, many 'section' wrappers in About Us page is using <div class="section"> instead of <section> element
- Compare e.g. spacing on [About Us](https://womencodingcommunity.com/about) on [WCC code of conduct](https://womencodingcommunity.com/code-of-conduct) pages:

<img width="1728" alt="Screenshot 2024-05-10 at 13 52 50" src="https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/26842896/de237fa1-ed3f-441f-9ef8-490d6e04b827">

### Reproducibility
- [ ] The bug is reproducible consistently.
- [X] The bug is sporadic or occurs randomly.

### Impact
- [ ] The bug affects critical functionality.
- [X] The bug affects minor functionality.
- [ ] The bug is a cosmetic issue.

### Proposed Solution (optional)
- Fix pages to have a more consistent and semantic HTML structure throughout the site, i.e. use appropriate elements like section, article, header, footer, etc., according to their intended semantic meaning.
